source/features/clean-archived-issue.css

```diff
@@ -0,0 +1,3 @@
+.comment-form-head.tabnav {
```

Comment:
This selects all the navigation, it has no reference to archived issues

Reply:
Thanks, I will check the MDN docs: https://developer.mozilla.org/en-US/docs/Web/CSS/:has